circle.yml

```diff
@@ -18,6 +18,12 @@ jobs:
           POSTGRES_USER: vapor_username
           POSTGRES_DB: vapor_database
           POSTGRES_PASSWORD: vapor_password
+      - image: scenecheck/postgres-ssl:latest
```

Comment:
Should we host this on Vapor's dockerhub? Unless you are happy to maintain this going forward.

Reply:
This should hopefully require very little maintenance - it will always use the latest postgres version. The image is on Docker Hub so it shouldn't go away anytime soon.